debian/changelog

```diff
@@ -1,3 +1,10 @@
+chromium-browser (79.0.3945.130-0endless1) eos; urgency=medium
```

Comment:
There is no need to bump the changelog here as we try to keep it in sync with [upstream xenial-stable](https://code.launchpad.net/~chromium-team/chromium-browser/xenial-stable), but it won't hurt and we can drop the entries in our next rebase (we already have a custom entry as xenial-stable doesn't have .130 yet anyway).